src/client/package.json

```diff
@@ -50,7 +50,7 @@
     "build": "yarn manage:translations && node ./scripts/build-non-split.js",
     "postbuild": "rm -rf ../extension/react && mv build ../extension/react",
     "manage:translations": "node scripts/manageTranslations.js",
-    "test": "react-scripts test",
+    "test": "react-scripts test --coverage",
```

Comment:
That's nice, we have access to the coverage!

Reply:
Yea, it's kinda misleading though since its already at 17% lol